bazel/grpc_deps.bzl

```diff
@@ -100,6 +100,16 @@ def grpc_deps():
         actual = "@com_google_googletest//:gtest",
     )
 
+    native.bind(
```

Comment:
If possible, we avoid new uses of `bind`. It has been discouraged for a while. We only keep these usages for backward-compatibility.

Reply:
what's the alternative? just nuke it?

Reply:
Yes. And reference it using the full name.